src/nvim/buffer.c

```diff
@@ -542,7 +542,7 @@ bool close_buffer(win_T *win, buf_T *buf, int action, bool abort_if_last)
 
   /*
    * Free all things allocated for this buffer.
-   * Also calls the "BufDelete" autocommands when del_buf is TRUE.
+   * Also calls the "BufDelete" autocommands when del_buf is true.
    */
```

Comment:
I think this comment needs to be changed to a `//`.

Reply:
That's what I thought too. clint didn't give a warning so I just skipped it. Fixing it now.